### DISCUSSION TITLE
client.get_user discards returned fields

### DISCUSSION THREAD

**Discussion by NimSed** [tweepy/tweepy#1992] [Questions] [answered]
Hi there.
The below call
`response = client.get_user(username='jack', user_fields=['description'])`
returns this:
`Response(data=<User id=12 name=jack username=jack>, includes={}, errors=[], meta={})`
As you see only id, name and username are there, there's no "description", and "includes" is empty (and I have triple-checked, it is not just missing in the printed version). 
Now here are the logs:
`
Headers: {'date': 'Wed, 26 Oct 2022 22:38:43 UTC', 
XXXXXXXXX
'x-response-time': '25', XXXXXXX
Content: b'{"data":{"description":"#bitcoin","username":"jack","id":"12","name":"jack"}}'
`
As you see the description field _is_ in the response. But it is discarded, apparently in: 
https://github.com/tweepy/tweepy/blob/48ebdb7face9188a352bfa82c616b20c6de12475/tweepy/client.py#L148

Same thing happens with "profile_image_url" and alike. What am I missing?

**Comment by Harmon758** [comment, marked as answer]
> Why am I not getting expansions or fields data with API v2 using [`Client`](https://tweepy.readthedocs.io/en/v4.11.0/client.html#tweepy.Client)?
> 
> If you are simply printing the objects and looking at that output, the string representations of API v2 models/objects only include the default fields that are guaranteed to exist.
> 
> The objects themselves still include the relevant data, which you can access as attributes or by subscription.
> 
> There’s also a `data` attribute/key that provides the entire data dictionary.

https://tweepy.readthedocs.io/en/v4.11.0/faq.html#why-am-i-not-getting-expansions-or-fields-data-with-api-v2-using-client

> it is not just missing in the printed version

`includes` is empty because you're not requesting any expansions.
As for `description`, I'm unable to reproduce this:

```pycon
>>> response = client.get_user(username='jack', user_fields=['description'])
>>> response.data.description
'#bitcoin'
```

There is no "discarding" being done.
Feel free to create a new issue if you have a reproducible example (MCVE/SSCCE) of missing fields.
